src/Test/Proxy/NginxProxy.php

```diff
@@ -15,7 +15,7 @@ class NginxProxy extends Abstractproxy
 {
     protected $binary = 'nginx';
     protected $configFile;
-    protected $port = 8080;
+    protected $port = 8088;
```

Comment:
hm, changing this default is a BC break. we should rather change the doc if it is incorrect.